.github/workflows/main.yaml

```diff
@@ -400,41 +399,26 @@ jobs:
       - name: Setup Environment
         id: setup
         run: |
+          # Setup Environment
+
           $Target='${{ matrix.target }}'
-          $ArtifactName="deps-windows-${Target}-${{ github.sha }}"
+          $ArtifactName="deps-windows-${Target}-${{ needs.pre-checks.outputs.shortHash }}"
           $FileName="windows-deps-$(Get-Date -Format 'yyyy-MM-dd')-${Target}.zip"
 
           "artifactName=${ArtifactName}" >> $env:GITHUB_OUTPUT
           "artifactFileName=${FileName}" >> $env:GITHUB_OUTPUT
 
-      - name: 'Check for GitHub Labels'
-        id: seekingTesters
-        if: ${{ github.event_name == 'pull_request' }}
-        run: |
-          $LabelFound = try {
-            $Params = @{
-              Authentication = 'Bearer'
-              Token = (ConvertTo-SecureString '${{ secrets.GITHUB_TOKEN }}' -AsPlainText)
-              Uri = '${{ github.event.pull_request.url }}'
-              UseBasicParsing = $true
-            }
-
-            (Invoke-RestMethod @Params).labels.name.contains("Seeking Testers")
-          } catch {
-            $false
-          }
-
-          "found=$(([string]${LabelFound}).ToLower())" >> $env:GITHUB_OUTPUT
-
-      - name: Build ntv2 debug
+      - name: Build ntv2 Debug
+        if: matrix.target == 'x64'
         shell: pwsh
         run: |
+          # Build ntv2 Debug
+
           $Params = @{
             Target = '${{ matrix.target }}'
             Configuration = 'Debug'
             Dependencies = 'ntv2'
           }
-          if ( '${{ matrix.type }}' -eq 'shared' ) { $Params += @{Shared = $true} }
```

Comment:
What's the purpose of removing this?

Reply:
The NTV2 SDK cannot be built as a shared library. Thus checking for the shared workflow variable is superfluous.